js/tests/vendor/jqueryloader.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  var path = '../../assets/js/vendor/jquery-slim.min.js'
```

Comment:
Maybe we don't need this new file at all? It's a small snippet so perhaps we could have it in the HTML file?

Reply:
yep sure :+1: